markdown/developers/graphic_design.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Graphic Design
+subtitle: Templates, common images, and design-related content for nf-core.
+---
+
+nf-core has a variety of recognisable images and other graphic designs for the community project. To help facilitate the identification of nf-core pipelines and related material, we provide here a variety of files that can be used for all nf-core-related content, e.g. pipeline logos, presentations, or workflow diagrams.
+
+All files are released under various forms of 'open' licenses (see each item), meaning you can re-use and re-model (sometimes with attribution) as you see fit.
+
+## Logo
+
+All information for the nf-core logo can be seen at the [nf-core/logos GitHub repository](https://github.com/nf-core/logos).
+
+## Fonts
+
+The font(s) typically used in nf-core graphic design (particularly logos) is [Maven Pro](https://fonts.google.com/specimen/Maven+Pro), which is released under a Open Font License (see link for details).
+
+The exact variants of the fonts can also be seen at the [nf-core/logos GitHub repository](https://github.com/nf-core/logos).
+
+## Colours
+
+The nf-core green colour is as follows:
+
+* RGB: 36,176,100
+* Hex: #24B064.
+
+## Presentation Templates
+
+nf-core regularly host [community events](https://nf-co.re/events), such as hackathons, and encourage training the wider bioinformatics community in creating nextflow and nf-core pipelines.s
+
+If you wish to introduce or run training sessions to your group or network, you can access a Google Doc template **here**.
```

Comment:
```suggestion
<!--
If you wish to introduce or run training sessions to your group or network, you can access a Google Doc template **here**.
-->
```

Maybe can link to `usage/nf_core_tutorial`  here?

Reply:
I could, but AFAICS this is just slides (either the HTML or PDF), not a template. Could you provide a template/original version someone can base their own tutorials off of? Or just basic image files from the tutorial that you think would be useful for others.